sql/npc_list.sql

```diff
@@ -12506,7 +12506,7 @@ INSERT INTO `npc_list` VALUES (17207846,'Couvoullie','Couvoullie',133,460.533,8.
 INSERT INTO `npc_list` VALUES (17207847,'blank','     ',129,461.799,8.909,-156.580,1,50,50,0,0,0,2,27,0x0000560000000000000000000000000000000000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17207848,'Luck_Rune','Luck Rune',90,-362.167,-12.199,157.158,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17207849,'Cavernous_Maw','Cavernous Maw',128,-45.147,0.056,436.919,32769,40,40,0,0,96,0,3,0x0000130900000000000000000000000000000000,2,'WOTG',1);
--- NC: INSERT INTO `npc_list` VALUES (17207850,'NOT_CAPTURED','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
+INSERT INTO `npc_list` VALUES (17207850,'no_name_0','     ',122,-366.262,-16.000,325.967,32769,50,50,0,0,96,0,27,0x0000160900000000000000000000000000000000,34,NULL,1);
```

Comment:
I'd like if our "DSP name" was more descriptive (and the filename too) instead of `no_name`!

(Can keep the packet name blank.)

Reply:
changed it to Lycopodium_Teleport
and for the one non related to Lycopodiums i renamed it no_name_0 as I do not know what it does as its not scripted.

Reply:
thankfully we never see the name in game or act on it in server other than to get the file name - or we'd see it get truncated and bug out because Lycopodium_Teleport is to long for the entity data to hold. Try and stick to 15 or less in the future but this should be fine here. (we really oughta document the limitations somewhere..)

Reply:
uh..I'm assuming you tested after that change and your script loads/works instead of looking for Lycopodium_Tel.lua

Reply:
coz I just assumed it'll find long script names still, >.>

Reply:
ya know actually lets just name all 3 (they are in diff zones, no need to number them!) "Lyco_sparkles". Sound good?

Reply:
@TeoTwawki  changed it to Lyco_Sparkles (I personally liked it as no_name lol)